docs/hash.adoc

```diff
@@ -200,17 +200,26 @@ link:
     content: https://www.iso.org/obp/ui/#!iso:std:53798:en
   - type: rss
     content: https://www.iso.org/contents/data/standard/05/37/53798.detail.rss
-# relations are an array of type, bibitem, and bib_locality.
+# relations are an array of type, bibitem, locality, and source_locality.
```

Comment:
This document needs to be eliminated: this document has been moved to https://github.com/relaton/relaton.com/blob/master/_specs/relaton-yaml.adoc . Please make all changes there.